fixity.rst

```diff
@@ -4,37 +4,38 @@
 Fixity
 ======
 
-The Archivematica Storage Service exposes an `API endpoint`_ that allows users
-to check fixity on individual AIPs. The result of a fixity check is written
-back to the storage service and shown in the packages tab of the user-interface
-in the Fixity Date and Fixity Status columns:
+Checking fixity is the process of ensuring that digital files have not changed
+unexpectedly over time. The Archivematica Storage Service exposes an API
+endpoint that enables users to check the fixity of individual AIPs in storage.
+Users can access the API endpoint directly or by using the Fixity application
+developed by Artefactual.
 
-.. image:: images/fixity_packages.*
-   :align: center
-   :width: 80%
-   :alt: The packages tab in the Storage Service showing Fixity Date and Fixity Status.
+The fixity API endpoint validates the contents of the AIP against the checksums
+and payload oxum listed in the AIP's `Bagit files`_. This can tell users if
+files have been modified, deleted, or added to the AIP.
```

Comment:
I wonder if it would be any more clear to state that the process verifies the bag, in addition to the explanation you've provided. At least for those familiar already with bags.

Reply:
Good rec, I'll clarify a little more